docs/modules/ROOT/pages/getting-started/relationships-and-complex-filtering.adoc

```diff
@@ -0,0 +1,235 @@
+= Relationships and complex filtering
+
+In a Graph database such as Neo4j, Relationships are as, if not more, important than Nodes. In this tutorial, you'll learn how to add relationships to the Cypher query from the xref:getting-started/filters-and-projections.adoc[previous tutorial]. 
```

Comment:
This sentence sounds a little bit weird in my mouth, but I'm not a native speaker. The suggestion of the part that feels weird to me:
```suggestion
In a Graph database such as Neo4j, Relationships are as important as Nodes, if not more. In this tutorial, you'll learn how to add relationships to the Cypher query from the xref:getting-started/filters-and-projections.adoc[previous tutorial]. 
```